common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -210,17 +214,17 @@ def test_submit_blank_answer(self):
             lambda: self.problem_page.problem_name == self.problem_name,
             "Make sure the correct problem is on the page"
         )
-        # Leave the problem unchanged and click check.
-        self.assertNotIn('is-disabled', self.problem_page.q(css='div.problem button.check').attrs('class')[0])
-        self.problem_page.click_check()
+        # Leave the problem unchanged and click submit.
+        self.assertTrue(self.problem_page.q(css='.problem .submit').attrs('disabled'))
+        self.problem_page.click_submit()
```

Comment:
How does this test work? Given that the submit button is disabled, I don't understand how it is possible to click Submit and then see an incorrect status.